languages/elixir/exercises/concept/dna-encoding/.docs/instructions.md

```diff
@@ -1,12 +1,12 @@
 In your DNA research lab, you have been working through various ways to compress your research data to save storage space. One teammate suggests converting the DNA data to a binary representation:
 
-| Nucleic Acid | Code |
-| ------------ | ---- |
-| ' ' (a gap)  | 0000 |
```

Comment:
`' '` renders as `''` on the website

Reply:
Is this a bug on the website?

Reply:
I wouldn't say so. It's not in a `<pre>` element so I wouldn't expect all white space to be preserved exactly. It just... what happens by default in HTML 🤷 

![Screen Shot 2021-01-15 at 08 15 30](https://user-images.githubusercontent.com/7852553/104693624-47c03380-570a-11eb-86bc-47f4239ed6cb.png)

Reply:
Ah, I see. You could potentially add it as a <pre> element? (ie, ` `). That would work on the website but look odd in the file when read as Markdown I guess. Your existing solution of saying a space probably the best. I wonder if it might be clearer as `[a space]` rather than `a space`? (Sorry for the driveby comments!)

Reply:
I'll leave it as `a space` because `[a space]` looks like a broken markdown link to me.